pkg/mount/magic.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2014-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mount
+
+import "fmt"
+
+// This code does not use a map. Why?
```

Comment:
What is the magic? Is it the first few bytes of the block device?

Perhaps this package would be a better place: https://github.com/u-root/u-root/blob/master/pkg/mount/block/blockdev.go#L5

Reply:
fsmagic is the standard magic at the start of a partition.

Reply:
maybe we need a package fs?